tests/test_config.py

```diff
@@ -21,6 +21,24 @@ def test_lock_timeout(self):
     def test_key_prefix_default(self):
         self.assertEqual(self.conf.key_prefix, 'redbeat:')
 
+    def test_lock_key_default(self):
+        self.assertTrue("REDBEAT_LOCK_KEY" not in self.app.conf.keys())
+        self.assertTrue("redbeat_lock_key" not in self.app.conf.keys())
+        self.conf = RedBeatConfig(self.app)
+        self.assertEqual(self.conf.lock_key, 'redbeat::lock')
+
+    def test_disable_lock_key_4(self):
+        self.app.conf.redbeat_lock_key = None
+        self.assertTrue("redbeat_lock_key" in self.app.conf.keys())
+        self.conf = RedBeatConfig(self.app)
+        self.assertEqual(self.conf.lock_key, None)
+
+    def test_disable_lock_key_3(self):
+        self.app.conf.REDBEAT_LOCK_KEY = None
+        self.assertTrue("REDBEAT_LOCK_KEY" in self.app.conf.keys())
+        self.conf = RedBeatConfig(self.app)
+        self.assertEqual(self.conf.lock_key, None)
+
```

Comment:
Should there be a ```test_custom_lock_key()``` or is that covered elsewhere somehow?

Reply:
Yup.  I'll add a test for a custom lock key.